ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -140,36 +137,31 @@ QtObject {
     }
 
     function createCommunityCategory(communityId, categoryName, channels) {
-        // Not Refactored Yet
-//        chatsModelInst.communities.createCommunityCategory(communityId, categoryName, channels);
+        communitiesModuleInst.createCommunityCategory(communityId, categoryName, channels);
     }
 
     function editCommunityCategory(communityId, categoryId, categoryName, channels) {
-        // Not Refactored Yet
-//        chatsModelInst.communities.editCommunityCategory(communityId, categoryId, categoryName, channels);
+        communitiesModuleInst.editCommunityCategory(communityId, categoryId, categoryName, channels);
     }
 
-    function deleteCommunityCategory(categoryId) {
-        // Not Refactored Yet
-//        chatsModelInst.communities.deleteCommunityCategory(chatsModelInst.communities.activeCommunity.id, categoryId);
+    function deleteCommunityCategory(communityId, categoryId) {
+        communitiesModuleInst.deleteCommunityCategory(communityId, categoryId);
     }
 
     function leaveCommunity(communityId) {
         communitiesModuleInst.leaveCommunity(communityId);
     }
 
     function setCommunityMuted(communityId, checked) {
-        // Not Refactored Yet
-//        chatsModelInst.communities.setCommunityMuted(communityId, checked);
+        communitiesModuleInst.setCommunityMuted(communityId, checked);
     }
 
-    function exportCommunity() {
-        // Not Refactored Yet
-//        return chatsModelInst.communities.exportCommunity();
+    function exportCommunity(communityId) {
+        return communitiesModuleInst.exportCommunity(communityId);
     }
 
     function createCommunityChannel(communityId, channelName, channelDescription) {
-       communitiesModuleInst.createCommunityChannel(communityId, channelName, channelDescription);
+        communitiesModuleInst.createCommunityChannel(communityId, channelName, channelDescription);
```

Comment:
@osmaczko `createCommunityChannel` should not be part of `communitiesModuleInst`.

Reply:
I only changed the indentation here. When I commented it out, the 'new channel' functionality stopped working. I will keep it as is.